src/components/NewCommentForm.tsx

```diff
@@ -114,7 +115,7 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
 
         <div className="control has-icons-left has-icons-right">
           <input
-            type="text"
+            type="email"
```

Comment:
Ensure that the email input field is validated correctly, as it uses `type="email"` which provides basic validation.